source/configuration_manual/mail_location/imapc/index.rst

```diff
@@ -74,11 +83,13 @@ Connection Settings
      emails in ``All Mails``). Add :dovecot_core:ref:`pop3_deleted_flag` to
      mails that don't exist in POP3 server.
 
-   ``modseq``
+   ``no-modseq``
 
-     Access ``MODSEQ`` and ``HIGHESTMODSEQ``
+     Disable access to ``MODSEQ`` and ``HIGHESTMODSEQ`` which is available by
+     default.
```

Comment:
Disable access to ``MODSEQ`` and ``HIGHESTMODSEQ`` fields. By default these fields are available if the remote server advertises the CONDSTORE or the QRESYNC capability.